packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -457,6 +488,31 @@ export const UIModeView: React.FC<{}> = ({
           requestedCollapseAllCount={collapseAllCount}
           setFilterText={setFilterText}
         />
+        <Toolbar noShadow={true} noMinHeight={true} className='settings-toolbar' onClick={() => setTestingOptionsVisible(!testingOptionsVisible)}>
+          <span
+            className={`codicon codicon-${testingOptionsVisible ? 'chevron-down' : 'chevron-right'}`}
+            style={{ marginLeft: 5 }}
+            title={testingOptionsVisible ? 'Hide Testing Options' : 'Show Testing Options'}
+          />
+          <div className='section-title'>Testing Options</div>
+        </Toolbar>
+        {testingOptionsVisible && <SettingsView settings={[
```

Comment:
If we had an abstracted `ToolbarSection`, this part would probably be passed as a child to it. That'd make the nesting a little easier to understand on this